flask/Flask_GeoAPI_setup.md

```diff
@@ -0,0 +1,649 @@
+        required=True, type="Array",
+        default=[[13.4197998046875, 52.52624809700062],
+                 [13.387527465820312, 52.53084314728766],
+                 [13.366928100585938, 52.50535544522142],
+                 [13.419113159179688, 52.501175722709434],
+                 [13.4197998046875, 52.52624809700062]]
+     )
+})
+
+polygon_feature = api.model('PolygonFeature', {
+    'type': fields.String(default="Feature", require=True),
+    'geometry': fields.Nested(polygon, required=True)
+})
+
+linestring = api.model('LineStringGeometry', {
+    'type': fields.String(required=True, default="LineString"),
+    'coordinates': fields.List(
+        fields.List(fields.Float, required=True, type="Array"),
+        required=True,
+        type="Array",
+        default=[[13.420143127441406, 52.515594085869914],
+                 [13.421173095703125, 52.50535544522142],
+                 [13.421173095703125, 52.49532344352079]]
+     )
+})
+
+linestring_feature = api.model('LineStringFeature', {
+    'type': fields.String(default="Feature", require=True),
+    'geometry': fields.Nested(linestring, required=True)
+})
+
+point = api.model('PointGeometry', {
+    'type': fields.String(required=True, default="Point"),
+    'coordinates': fields.List(
+        fields.List(fields.Float, required=True, type="Array"),
+        required=True,
+        type="Array",
+        default=[13.421173095703125, 52.49532344352079]
+     )
+})
+
+point_feature = api.model('PointFeature', {
+    'type': fields.String(default="Feature", require=True),
+    'geometry': fields.Nested(point, required=True)
+})
+```
+
+Just take a moment and understand how the models are generated. You'll use the Polygon and LineString features in this tutorial. The remaining models are just exemplary and should provide you a base on which you can later build (hopefully) your own implementations on.
+
+### Calculate the area of a polygon
+
+As the first GeoAPI request, you'll implement a simple calculation to get the area of a Polygon into `app/modules/geoapi/resources.py`. The function is going to accept a GeoJSON object via a POST endpoint, deserialize it following the `PolygonFeature` model you provided above, calculate its area in the [Robinson projection](https://spatialreference.org/ref/esri/world-robinson/) and return the value.
+
+Now go ahead and implement your first endpoint function and paste this code below the models:
+
+```python
+@api.route('/polygon/area/')
+class PolygonArea(Resource):
+    """
+    Return the area of a polygon in m²
+    """
+
+    @api.expect(polygon_feature, validate=True)
+    @api.doc(id='polygon_area')
+    def post(self):
+        """
+         Return the area of a polygon in m²
+        """
+
+        try:
+            projection = partial(
+                pyproj.transform,
+                pyproj.Proj(init='epsg:4326'),
+                pyproj.Proj('+proj=robin +lon_0=0 +x_0=0 +y_0=0 +ellps=WGS84 +datum=WGS84 +units=m +no_defs')
```

Comment:
All other endpoints could use Robinson. This is the only endpoint that shouldn't use it.. Robinson is NOT an equal-area projection. Please change to e.g. https://spatialreference.org/ref/epsg/3035/

Reply:
I changed it